### DISCUSSION TITLE
Give the ability to update Microsoft Openjdk/Openjre through the Windows/Microsoft Update Process

### DISCUSSION THREAD

**Discussion by steskalja** [microsoft/openjdk#30] [Q&A] [answered]
When will it be added to Windows10 and Windows Server so that it can be updated through windows updates? 

The reason for is to allow for better patching of the Openjdk to stay up to the latest version. This would be the easiest method to do these updates in a timely manner.

**Comment by karianna** [comment, marked as answer]
Also see #13
